sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2Suite.scala

```diff
@@ -905,6 +908,13 @@ object ColumnarReaderFactory extends PartitionReaderFactory {
 
   override def supportColumnarReads(partition: InputPartition): Boolean = true
 
+  override def getVectorTypes: Optional[java.lang.Iterable[String]] = {
+    val data: Iterable[String] = Iterable.fill(2)(
+        classOf[OnHeapColumnVector].getName
+    )
```

Comment:
Oh, that's also very nice (re my previous comment to use `Collections.nCopies`).

Can you rename `data`?

Reply:
> Oh, that's also very nice (re my previous comment to use `Collections.nCopies`).
> 
> Can you rename `data`?

Thank you, I will try to modify the code as you say